test/resizing.jl

```diff
@@ -144,5 +144,20 @@ end
             @test ndims(R) == 0
             @test !(R === A)
         end
+
+    @testset "Interpolation" begin
+        img = rand(16,16)
+        out = ImageTransformations.imresize(img,(128,128),method=Lanczos4OpenCV())
+        @test size(out) == (128,128)
+
+        out = ImageTransformations.imresize(img,(128,128),method=Linear())
```

Comment:
We need to test the default behavior:

```julia
imresize(img, (128,128), method=Linear()) == imresize(img, (128,128))
```

`imresize` is exported so no need to write `ImageTransformations.imresize`.

Reply:
Format isn't a critical issue but also worths noting:

```diff
- imresize(img, (128,128), method=Linear())
+ imresize(img, (128,128), method=Linear())
```

See also the blue style: https://github.com/invenia/BlueStyle#whitespace